kinto/core/resource/__init__.py

```diff
@@ -599,7 +613,7 @@ def process_record(self, new, old=None):
 
         return new
 
-    def apply_changes(self, record, changes):
+    def apply_changes(self, record, changes, operations=None):
```

Comment:
Sorry but I don't get why we couldn't use the `changes` object to pass the operations here?